test/views/html/InflationProofSpec.scala

```diff
@@ -47,7 +47,7 @@ class InflationProofSpec extends GmpViewSpec {
         .verifying(messages("gmp.error.month.invalid"), x => checkMonthRange(x.month))
         .verifying(messages("gmp.error.year.invalid.format"), x => checkYearLength(x.year))
       ,
-      "revaluate" -> optional(text).verifying(messages("gmp.error.reason.mandatory"),{_.isDefined})
+      "revaluate" -> optional(text).verifying(messages("revaluation.revaluate.error.required"),{_.isDefined})
```

Comment:
the message key we should use here is `revaluate.error.required` (line 487 of message file) - if the other key exists in the messages we should probably delete it